modules/telestion-application/src/main/java/de/wuespace/telestion/application/loader/NoConfiguration.java

```diff
@@ -0,0 +1,13 @@
+package de.wuespace.telestion.application.loader;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/**
+ * Use this type of configuration to indicate that your {@link TelestionLoader} doesn't accept any configuration.
+ *
+ * @author Pablo Klaschka (@pklaschka), Cedric Boes (@cb0s), Ludwig Richter (@fussel178)
+ * @see TelestionLoader#getConfig()
+ */
+@JsonIgnoreProperties(ignoreUnknown = false)
+public record NoConfiguration() implements LoaderConfiguration {
+}
```

Comment:
I think this doc could use an example of how to "plug" this configuration type into the loader (i.e., just a simple `Loader` class with this put into the generic field)